pull-requests.md

```diff
@@ -90,6 +90,9 @@ While looking at the PR you should remember to focus on the changes in the PR.
 
 It is tempting, if there are general problems with the code being examined, to start requesting changes to improve the pre-existing code. Small improvements are sometimes fine, however large amounts of refactoring pre-existing code should not be done via comments in a PR. If such work is needed it should have its own pull request (and potentially go into team planning as a discrete piece of work).
 
+## Make actionable comments
+Keep in mind the person who raised the PR - especially if they are newer to the codebase or company you may need to be clear what action they should take and how important it is.  e.g. "optional: consider renaming to cancellationDate" or other prefixes such as nitpick/blocker/question/future/note.
```

Comment:
❤️ 

I tend to use "required:" instead of "blocker:". It allows me to approve the PR while also letting the author know which comment is required to be actioned before merge.